orbotservice/src/main/java/org/torproject/android/service/OrbotService.java

```diff
@@ -316,6 +314,11 @@ private void stopTorAsync() {
             sendCallbackStatus(STATUS_STOPPING);
             sendCallbackLogMessage(getString(R.string.status_shutting_down));
 
+            String bridgeList = Prefs.getBridgesList();
+            boolean useIPtProxy = bridgeList.contains("obfs3")|| bridgeList.contains("obfs4")||bridgeList.contains("meek");
```

Comment:
This variable declaration is written out twice in OrbotService. It should be a static function